tikv-client/src/main/java/com/pingcap/tikv/codec/MyDecimal.java

```diff
@@ -268,21 +267,22 @@ private int digitsToWords(int digits) {
    * @param size is word size which can be used in switch statement.
    * @param start start indicates the where start to read.
    */
-  private int readWord(int[] b, int size, int start) {
+  @VisibleForTesting
+  public static int readWord(int[] b, int size, int start) {
     int x = 0;
     switch (size) {
       case 1:
-        x = b[start];
+        x = (byte)b[start];
```

Comment:
same reason.

Reply:
In java byte is signed. b[start] is unsigned in fact since we read as unsigned byte.
So actually convert to byte will cause it to be signed. It's not trivial.